src/olympia/blocklist/models.py

```diff
@@ -325,3 +325,24 @@ def save_to_blocks(self):
             self.save()
 
         return blocks
+
+
+class KintoImport(ModelBase):
+    OUTCOME_INCOMPLETE = 0
+    OUTCOME_BADGUID = 1
+    OUTCOME_NOTFIREFOX = 2
+    OUTCOME_BLOCK = 3
+    OUTCOME_REGEXBLOCKS = 4
+    OUTCOME_NOMATCH = 5
+    OUTCOMES = {
+        OUTCOME_INCOMPLETE: 'Incomplete',
+        OUTCOME_BADGUID: 'Missing GUID',
```

Comment:
If the meaning is "Missing GUID", would a better name for the key be `OUTCOME_MISSINGGUID`? `BADGUID` implies to me that it could be invalid and not just missing.

Reply:
https://github.com/mozilla/addons-server/pull/13274/commits/05b0c2536b4bf1b1f310a482ad84345acf39477d